rca/programmes/serializers.py

```diff
@@ -7,3 +8,28 @@ class DegreeLevelSerializer(serializers.ModelSerializer):
     class Meta:
         model = DegreeLevel
         fields = ("id", "title")
+
+
+class RelatedSubjectSerializer(Field):
+    def to_representation(self, value):
+        for v in value.all():
+            subjects = []
+            for v in value.all():
+                subject = {}
+                subject["id"] = v.subject.id
+                subject["title"] = v.subject.title
+                subject["description"] = v.subject.description
+                subjects.append(subject)
+        return subjects
+
+
+class RelatedSchoolSerializer(Field):
```

Comment:
This could be a model serializer.

Reply:
`APIField("subjects", serializer=subject_serializer(many=True)),` This is a list, right?

Reply:
removed